action-server/covidflow/setup_structlog.py

```diff
@@ -36,9 +37,12 @@ def setup_structlog(
         default = "pretty" if sys.stdout.isatty() else "json"
         log_style = environ.get("log_style", default)
 
-    if not logging.root.handlers:
-        handler = logging.StreamHandler()
-        logging.root.handlers.append(handler)
+    handler = logging.StreamHandler()
+    handler.setFormatter(CustomJsonFormatter())
+    logging.root.handlers.append(handler)
+    sanic.log.access_logger.handlers.append(handler)
+    sanic.log.error_logger.handlers.append(handler)
+    sanic.log.logger.handlers.append(handler)
```

Comment:
is it really necessary to remove the `if` statement? I think the purpose is to avoid adding multiple handlers in the case `setup_structlog` is called multiple times... (?)

Reply:
It shouldn't be called multiple times, however I think the root logger is always present so I re-added the condition.

Reply:
_shouldn't_ :)